tools/clang/test/HLSL/cpp-errors-hv2015.hlsl

```diff
@@ -51,10 +51,6 @@ __real g___real; // expected-error {{expected unqualified-id}}
 __FUNCTION__ g___FUNCTION; // expected-error {{expected unqualified-id}}
 __PRETTY__ g___PRETTY; // expected-error {{unknown type name '__PRETTY__'}}
 
-struct s_with_bitfield {
-  int f_bitfield : 3; // expected-error {{bitfields are not supported in HLSL}}
-};
-
```

Comment:
We need to keep this failure for 2015 as it's the fxc compatibility target. 2018 is fuzzier. I don't anticipate any conflicts with existing code, so maybe it's okay to enable there, but since we need to check the version anyway, maybe we just check for 2021

Reply:
Should I revert cpp-errors as well, or specify -HV 2021?